boards/qtpy-esp32s2/definition.json

```diff
@@ -104,26 +118,44 @@
             {
                 "name":"A9",
                 "displayName":"A2",
-                "direction":"INPUT",
-                "dataType":"int16"
+                "dataType":"int16",
+                "hasPWM":true,
+                "hasServo":true
             },
             {
                 "name":"A8",
                 "displayName":"A3",
-                "direction":"INPUT",
-                "dataType":"int16"
+                "dataType":"int16",
+                "hasPWM":true,
+                "hasServo":true
             },
             {
                 "name":"A7",
                 "displayName":"SDA",
-                "direction":"INPUT",
-                "dataType":"int16"
+                "dataType":"int16",
+                "hasPWM":true,
+                "hasServo":true
             },
             {
                 "name":"A6",
                 "displayName":"SCL",
-                "direction":"INPUT",
-                "dataType":"int16"
+                "dataType":"int16",
+                "hasPWM":true,
+                "hasServo":true
+            },
+            {
+                "name":"A5",
+                "displayName":"A5 (UART TX)",
```

Comment:
why is this not `uartTx`?

Reply:
Not part of the analog pins schema, should it be?